test/testlib/minikube_utilities.go

```diff
@@ -544,13 +544,27 @@ func GetDatabaseIncarnation(t *testing.T, namespace string, podName string, data
 	return nil
 }
 
+func GetRestartCount(t *testing.T, podName string, options *k8s.KubectlOptions) int32 {
```

Comment:
we already have `GetPodRestartCount`. How is this different?

Reply:
It is different in that it takes the options from the caller directly.
However, my code doesn't use that to get different behaviour.

I can check the history to see if I just missed the existing GetPodRestartCount, or whether it simply did not exist when I wrote mine, and has been implemented since.